doc/user/inputs/settings.rst

```diff
@@ -61,3 +61,34 @@ to update this setting.
 
 Other settings on this tab may need to be updated depending on your computational environment.
 Talk to your system admins to determine which settings are best.
+
+Some special settings
+=====================
+A few settings warrant additional discussion.
+
+.. _detail-assems:
+
+Detail assemblies
+-----------------
+Many plugins perform more detailed analysis on certain regions of the reactor. Since the analyses
+often take longer, ARMI has a feature, called *detail assemblies* to help. Different plugins
+may treat detail assemblies differently, so it's important to read the plugin documentation
+as well. For example, a depletion plugin may perform pin-level depletion and rotation analysis
+only on the detail assemblies. Or perhaps CFD thermal/hydraulics will be run on detail assemblies,
+while subchannel T/H is run on the others. 
+
+Detail assemblies are specified by the user in a variety of ways, 
+through the GUI or the settings system.
+
+Detail Assembly Locations BOL
+    The ``detailAssemLocationsBOL`` setting is a list of assembly location strings 
+    (e.g. ``A4003`` for ring 4, position3). Assemblies that are in these locations at the 
+    beginning-of-life will be activated as detail assemblies.
+
+Detail assembly numbers
+    The ``detailAssemNums`` setting is a list of``assemNum``s that can be inferred from a previous
+    case and specified, regardless of when the assemblies enter the core. This is useful for
+    activating detailed treatment of assemblies that enter the core at a later cycle.
```

Comment:
I think "detail assemblies" is getting perhaps a little overgrown. It can be surprising when certain extra analyses are performed that aren't expected, and there is no way of controlling what is done and what isn't, given a collection of enabled plugins, without extra plugin-specific settings. Consider adding a note that we plan on re-thinking it at some point.

Reply:
adding note. Also, you have to use `\ s` to pluralize things like that apparently. good catch!